stdlib/2.7/codecs.pyi

```diff
@@ -115,9 +115,9 @@ class Codec:
         ...
 
 class IncrementalEncoder:
+    errors = ...  # type: str
     def __init__(self, errors: str = ...) -> None:
-        self.errors = errors
-    @abstractmethod
```

Comment:
`@abstractmethod` can actually be important, if the method really is supposed to be abstract. Why did you remove it?